src/chrome-launcher.ts

```diff
@@ -144,7 +144,7 @@ class Launcher {
   }
 
   private get flags() {
-    const flags = this.ignoreDefaultFlags ? [] : DEFAULT_FLAGS;
+    const flags = this.ignoreDefaultFlags ? [] : DEFAULT_FLAGS.slice();
```

Comment:
shouldn't we be `.slice`'ing the `defaultFlags` return on line 167?

Reply:
I did consider doing that - it's probably a good idea. It would certainly be safer.

I held off initially because it's not really necessary for any projects using TypeScript, and avoiding it would mean avoiding duplicating the array. But in retrospect, that's such a tiny micro-optimization that it's hardly worth considering. I'll update it.